docs/stylesheets/extra.css

```diff
@@ -3,6 +3,6 @@
     margin: 0 auto;
 }
 
-.header{
-	font-size: x-large;
+.header {
+    font-size: 24px;
```

Comment:
Check if this affected the rest of the sites

Reply:
does not affect